src/layouts/shortcodes/Button.jsx

```diff
@@ -1,14 +1,24 @@
-const Button = ({ href, style, rel, children }) => {
+import React from "react";
+
+const Button = ({ 
+  href = "#", 
+  style = "secondary", 
+  children = "Click Me", 
+  rel = "noopener noreferrer" 
+}) => {
+  const buttonStyles = {
+    primary: "bg-white text-black border border-black hover:bg-black hover:text-white",
```

Comment:
@SanjanaSogimatt 
this is fine. But it also assumes that primary and secondary buttons have certain colours.
Do you think we should finalize the colours and branding first? Editing these later is going to be a hassle.